data/scripts/network/cyclopedia_character.lua

```diff
@@ -0,0 +1,49 @@
+local BASIC_INFO = 0x00
+
+--[[
+	local GENERAL_STATS = 0x01
+	local COMBAT_STATS = 0x02
+	local RECENT_DEATHS = 0x03
+	local RECENT_PVP_KILLS = 0x04
+	local ACHIEVEMENTS = 0x05
+	local ITEM_SUMMARY = 0x06
+	local APPEARANCES = 0x07
+	local STORE = 0x08
+	local INSPECTION = 0x09
+	local BADGES = 0x0A
+	local TITLES = 0x0B
+]]--
+
+local methods = {
+	[BASIC_INFO] = "sendBasicInfo",
```

Comment:
Move the `sendBasicInfo` function to this file, and refer to it via a reference instead, i.e.:

```lua
local function sendBasicInfo(self, msg)
    ...
end

local methods = {
	[BASIC_INFO] = sendBasicInfo,
    ...
}
```

then

```
local method = methods[type]
```

Also: better call it `handler`/`handlers` instead of `method`/`methods`

Reply:
I don't know exactly if that's what you asked for, if it's not enough I'll send new changes.

Reply:
It is, awesome